pandas/tests/reshape/test_reshape.py

```diff
@@ -462,7 +462,7 @@ def test_preserve_categorical_dtype(self):
             cidx = pd.CategoricalIndex(list("xyz"), ordered=ordered)
             midx = pd.MultiIndex(levels=[['a'], cidx],
                                  labels=[[0, 0], [0, 1]])
-            df = DataFrame([[10, 11]], index=midx)
+            df = DataFrame([[10, 11]] * 2, index=midx, columns=[0, 1])
```

Comment:
This ``columns=[0,1]`` is a typo. Waiting for any comments on the rest, then I will remove it.